.github/workflows/test.yml

```diff
@@ -34,6 +36,9 @@ jobs:
       - name: Run Ray tests
         run: |
           coverage run --parallel-mode -m pytest --system-name ray
+      - name: Run MPI tests
```

Comment:
Can we hold off on adding MPI testing until we have nightly builds? As-is, this will increase test time significantly.